ci/run.sh

```diff
@@ -52,3 +52,8 @@ gpuci_logger "Starting upload..."
 GPUCI_RETRY_MAX=5
 GPUCI_RETRY_SLEEP=120
 gpuci_retry docker push ${DOCKER_IMG}:${DOCKER_TAG}
+
+if [ "$TAG" = "0.18-cuda11.0-base-ubuntu18.04-py3.8" ]; then
```

Comment:
I think it should be `DOCKER_TAG`?

Reply:
yes! good catch. copy-paste error on my part. thanks for catching this.

Reply:
fixed in 4625244